### DISCUSSION TITLE
How to go to file from "find all references" view

### DISCUSSION THREAD

**Discussion by humblehacker** [zed-industries/zed#14893] [Feature Requests]
When I invoke "find all references", I'm taken to a nice view showing each file and the snippets containing the referenced identifier. How can I then jump from one of these snippets to the whole file with the cursor on the correct line?

**Comment by humblehacker** [comment]
I found the answer by button mashing - `alt-enter`, or if using vim, `G ␣`. Both mapped to `editor::OpenExcerpts`.

**Reply by hristogochev** [reply]
I was wondering the same thing, thank you so much! One thing to note is that when you are on MacOS and using the JetBrains Keymap, the mapping of `alt-enter` is not there by default and you have to add it.  e.g.: `"alt-enter": "editor::OpenExcerpts"`

**Reply by Tobbe** [reply]
I found this and was a little confused by "or if using vim, `G ␣`".
The key combo you should use is actually `g ␣`. I.e. lowercase 'g' plus 'space', not 'G'. Trying to type a capital 'G' ('shift' plus 'g') will make you jump to the end of the file.
(`G ␣` is actually what it says in the command palette as well, and it's equally confusing there)

**Reply by schdaniel20** [reply]
Mr. Tobbe, you are a legend. Your name will never be forgotten. Thank you!
